codegen-renderers/src/test/kotlin/io/vrap/codegen/languages/TestCodeGenerator.kt

```diff
@@ -42,6 +45,37 @@ class TestCodeGenerator {
         generatorComponent.generateFiles()
     }
 
+    @Test
+    fun generateJavaModelsSecond() {
+        val generatorConfig = CodeGeneratorConfig(basePackageName = "com.commercetools.importer", outputFolder = Paths.get("build/gensrc/java"))
+        val generatorModule = GeneratorModule(apiProvider, generatorConfig, JavaBaseTypes)
+        val generatorComponent = GeneratorComponent(generatorModule, JavaModelModuleSecond())
+        generatorComponent.generateFiles()
+    }
+
+    /**
+     * This test method uses code generator to generate interface and a class for simple-type.raml which is a part of the test-api.raml located in the resources
+     * folder. After the classes are generated, it checks if they are the same as the ones specified in SimpleType.txt and SimpleTypeImpl.txt.
+     */
+    @Test
+    fun compareGenerated() {
+        cleanGenTestFolder()
+        val testApiProvider = ApiProvider(Paths.get("src/test/resources/java/ramlTestFiles/test-api.raml"))
+        val generatorConfig = CodeGeneratorConfig(basePackageName = "com.commercetools.test", outputFolder = Paths.get("build/gensrc/java"))
+        val generatorModule = GeneratorModule(testApiProvider, generatorConfig, JavaBaseTypes)
+        val generatorComponent = GeneratorComponent(generatorModule, JavaModelModuleSecond())
+        generatorComponent.generateFiles()
+        val generatedSimpleTypeInterface = String(Files.readAllBytes(Paths.get("build/gensrc/java/com/commercetools/test/models/simpleTypes/SimpleType.java")))
+        val generatedSimleTypeClass = String(Files.readAllBytes(Paths.get("build/gensrc/java/com/commercetools/test/models/simpleTypes/SimpleTypeImpl.java")))
+
+        val correctSimpleTypeInterface = String(Files.readAllBytes(Paths.get("src/test/resources/java/ramlTestFiles/generated/SimpleType.txt")))
+        val correctSimpleTypeClass = String(Files.readAllBytes(Paths.get("src/test/resources/java/ramlTestFiles/generated/SimpleTypeImpl.txt")))
+
+        Assert.assertEquals(correctSimpleTypeClass.trim(), generatedSimleTypeClass.trim())
```

Comment:
I get what you want to achieve with `.trim()` here, but I would prefer to perform the assert without it. It makes it more ovious what we compare and the files just have to be exactly equal.